common/src/main/java/fr/openent/presences/enums/EventBusActions.java

```diff
@@ -0,0 +1,35 @@
+package fr.openent.presences.enums;
+
+public enum EventBusActions {
+
+    ADDFOLDER("addFolder"),
+    DELETE("delete"),
+    GETDOCUMENT("getDocument"),
+    LIST("list");
+
+    private final String action;
+
+    EventBusActions(String action) {
+        this.action = action;
+    }
+
+    public String action() {
+        return this.action;
+    }
+
+    public static enum EventBusAddresses {
```

Comment:
Modifier 'static' is redundant for inner enums 

la static est redondante, elle est déjà accessible via la class inner enum